frontend/src/v5/ui/controls/modal/formModal/formDialog.styles.ts

```diff
@@ -54,7 +54,7 @@ export const Title = styled(Typography).attrs({
 `;
 
 export const FormDialogContent = styled(DialogContent)`
-	margin: 20px 34px 43px;
+	margin: ${({ zeroMargin }) => (zeroMargin ? '0' : '20px 34px 43px')};
```

Comment:
This looks like an inconsitency in the dialog desing. Is it ok that we have margins in "new container" and not here @george-hadfield ? In the case that it is ok. Is this a different type of content in the modal that will be reused?